docs/dlx.md

```diff
@@ -137,13 +138,11 @@ their original routing keys.  This includes routing keys
 added by the `CC` and `BCC` headers
 (refer to [Sender-selected distribution](./sender-selected) for details about these two headers).
 
+### Dead-letter cycle
 
-It is possible to form a cycle of message dead-lettering.  For
-instance, this can happen when a queue "dead-letters"
-messages to the default exchange without specifying a
-dead-letter routing key. Messages in such cycles (that is,
-messages that reach the same queue twice) are
-dropped <em>if there was no rejection in the entire cycle</em>.
+It is possible to form a cycle of message dead-lettering where the same message reaches the same queue twice.
+For example, this can happen when a queue "dead-letters" messages to the default exchange without specifying a dead-letter routing key.
+To prevent automatic infinite message looping within RabbitMQ, RabbitMQ will detect a cycle and drop the message <em>if there was no rejection in the entire cycle</em>.
```

Comment:
Please use Markdown, not HTML. I know the previous content used `<em>` but now is the chance to fix it.